First Draft of Social Media Policy

laurabaines opened the pull request:
Read through this first draft and let me know what you think might need changing, adding or amending.

laurabaines committed:
Test update.

laurabaines committed:
WIP: First Draft of Social Media Policy

Sections include: Social Media as a School, Keeping Staff Safe & Complaints

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
-# Open Source Social Media Policy v0.1
+# Social Media Policy
```
I think we should add a version number to the document, which we can update each time we create a release, this will make it easier for people to know which version of the document they are reading.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+### General Rules and Restrictions
+
+#### Posting on Social Media
+* Content will be posted to official school pages only.
```
'page' is a facebook term, it should maybe something like account/profile/page

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+* Content will be posted to official school pages only.
+* There will be no content posted to a private page or closed group.
+* All posts are considered public, regardless of where they are posted, and should be treated as such.
+* The original post must be posted on a school page or account, but can then be shared by others from this place.
```
Maybe consider adjusting to something on the lines of 'all school derived content should be posted via official school accounts only, and then reshared/posted/retweeted by teachers/pupils and parents.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+* There will be no content posted to a private page or closed group.
+* All posts are considered public, regardless of where they are posted, and should be treated as such.
+* The original post must be posted on a school page or account, but can then be shared by others from this place.
+* Any photographs or videos taken within school grounds, or during school time, should only be shared to a school account. 
```
Do school trips etc.. need explicitly mentioning?

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+
+#### Photographs and Video
+* Media will be posted to school accounts only.
+* If the school keep a list of children who do not have parental consent to be online, this should be cross referenced before every post (and an audit trail of this kept for future reference).
```
'Must Be' not 'should'. - If the suggestion is an audit trail, in what form would this be? how could/should it be implemented.  - The Audit trail might need dropping in all fairness.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+
+#### Engaging with Parents and Pupils
+*	Employees will not engage with parents or pupils via a closed or private account.
+*	All interactions must happen publicly and through school official channels only.
```
I am not sure a parent complaining about how bad a school is should be public, maybe adjust to All interactions must happen though the official school accounts, all interactions are encourage to be public where feasible possible and suitable for that to happen.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+#### Engaging with Parents and Pupils
+*	Employees will not engage with parents or pupils via a closed or private account.
+*	All interactions must happen publicly and through school official channels only.
+*	Staff are encouraged to engage and reply where possible to increase engagement with others, except in the case where children are using a site which is not age-appropriate for them.
```
Caveat - only with the official school account?

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+
+#### Permissions
+*	Unless stated otherwise, all media content obtained by employees remains the employees content. Employees may grant the school access to the content for distribution on social networks.
+*	All media uploaded must be owned by the school before it can be uploaded.
```
All media must be owned by the person uploading it, or the organisation the person works for.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+*	Social Networks will be managed either through a single person or via a social media management solution in which all users will have their own log in and password. This is to ensure that full accountability exists.
+
+#### Permissions
+*	Unless stated otherwise, all media content obtained by employees remains the employees content. Employees may grant the school access to the content for distribution on social networks.
```
Unless otherwise stated in the employment contract all media and IP (intellectual property) remains property of the person that created the media. By uploading to the schools social network account they give permission to the school to use the media under the creative commons licence.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+
+#### Managing Social Networks
+*	We have assigned our social media specialist as __Mr Anonymous__.
+* We endeavour to ensure all content is moderated to ensure consistency and to ensure our posts paint the school in a positive light. 
```
Staff should ensure that content is checked by another member of the team before posting, to ensure the content is abiding by the details in this policy.

BenjaminGibbs commented on socialmedia.md:
```diff
@@ -1,20 +1,118 @@
+
+### General Rules for Employees
+*	Where possible, all social accounts which are personal to staff members must have two factor authentication on their account to avoid misuse by others.
+*	Passwords on personal accounts must remain private and not be shared.
```
and should be changed every 3 months.

BenjaminGibbs commented:
Not a bad start @laurabaines, I have made some comments. You should also go though it and adjust wording as suggested [here](https://tools.ietf.org/html/rfc2119) - and maybe add the definitions as an appendix at the bottom for clarity.

laurabaines committed:
WIP: Second Draft of Social Media Policy

Re-drafted whole document, refined terminology for 'shared' 'posted' and 'uploaded', added definitions to the beginning, ensured use of 'should' 'must' and 'may' is accurate.

laurabaines committed:
WIP: Added the Aims section

Added the aims section which outlines why schools use social media.

laurabaines committed:
WIP: Re-structured the Must, Should, May sections

Under each heading, I have restructured the Must, Should, May

laurabaines committed:
WIP: Corrected the Should headings

laurabaines committed:
WIP: Third Draft of Social Media Policy

Redrafted to include a whole restructure of content.

laurabaines committed:
WIP: Spell Checked

laurabaines committed:
WIP: Changed the custom fields to all blank / bold

Everywhere that someone needs to input their own information will not display as _Add Here_ in bold.

laurabaines committed:
WIP: Added a parental responsibilities section

Added a must, should and may section on parents.

laurabaines committed:
WIP: Taken Social Media Explanations to new doc

laurabaines committed:
WIP: New document with Social Media Explained

laurabaines committed:
WIP: Renamed

laurabaines committed:
WIP: New title

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+# Social Media Policy v0.1.0
+
```
Let's add in a "How to use" paragraph.
I am thinking on the lines of...

"This document is a ......., it is intended to be as light touch as possible allowing while still being safe online, this style would allow a school to use this policy as a base they could then lock down if needed."

While I know we may have that in another area of the repo I think that fact needs to be at the top of the document so people know they can lock down other stuff is needed.

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+* Youtube: __Add Here__
+
+## Usage and Best Practice
+
```
Lets add a paragraph in here.... I am thinking something on the lines of as a bit of an intro to the section as I think it needs on visually and for the narrative of the document if a stranger was reading it.
Maybe...
"To ensure we meet our aims, we have detailed usage and best practice on issues surrounding the use of social media within our school."

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+## Usage and Best Practice
+
+### Taking Photographs and Videos
+
```
Just a line...

"To be applied when capturing any content with a camera or microphone on any device."

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+* Employees may experiment with artistic photos, including close-ups and editing using image editing apps.
+
+### Posting to Social Media
+
```
Add a line,

"To be applied whenever posting anything on social media in professional capacity"

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+* Parents must treat all interactions with an official school account as public.
+
+#### Should (recommended)
+* Parents should not tag other children in photographs, but are free to tag their own if they wish.
```
I am not sure on this, should the school not have a policy on "zero tags" the reason is that from a management point of view checking who added a tag, then working out if it is a parent etc.. would be a nightmare and not manageable. 

Having Zero Tags makes it easy to manage and falls in line with all the effort we go to when taking a photo around not having one child, and not having names in the photo for example.

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+### Parental Responsibilites
+
+#### Must (required)
+* Parents must treat all interactions with an official school account as public.
```
Maybe adjust this to...

Parents must treat all interactions with an official school account as public as multiple employees of the school may have access to the social media account, for sensitive subjects it is recommend you speak to the school directly or email the corresponding person.

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+*	Any school content kept on a personal device must only be kept for the minimum amount of time feasible, up to a maximum of a week.
+
+#### Should (recommended)
+* Passwords to personal devices should have their pin/passwords changed every 3 months (minimum).
```
"at least every 3 months" ;)

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+*	Any personal devices used to hold content from school must be secured with a password or pin.
+*	Any school content kept on a personal device must only be kept for the minimum amount of time feasible, up to a maximum of a week.
+
+#### Should (recommended)
```
Add in something on the lines of

"Where possible should avoid using personal devices to capture content etc....."

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+
+##### Must (required)
+*	All media posted must be owned by the person uploading, or the organisation they work for, before it can be uploaded.
+*	Unless stated otherwise in an employment contract, all media and IP (intellectual property) remains the property of the person who created the media. By posting the content to a school account, permission is granted to the school to use the media under the Creative Commons License.
```
We should remove the CC bit for now, I am not sure the licence fits our needs after reading it in detail as you have to provided Attribution.

I would adjust that by posting they give school permission to share (copy and redistribute the material in any medium or format) and adapt (remix, transform, and build upon the material) the artwork.

This may need some more thought.

BenjaminGibbs commented on socialmediapolicy.md:
```diff
@@ -0,0 +1,155 @@
+*	Passwords must not be shared between employees.
+
+##### Should (recommended)
+*	Social Networks should be managed either through a single person or via a social media management solution in which all users will have their own log in and password.
```
Give me some examples? Tweet deck etc..?

BenjaminGibbs reviewed (commented):
I have added some changes/ideas.

The rational needs adding in to each of the points.

Also how does this cover a teacher going in to school taking photos of the classroom and posting online say in a private teaching group as we have seen? My thought is that is a area which is a grey area as classrooms have trays/name badges etc... even though it doesn't show a childs face you are still saying a child in the class is called "Joe Bloggs"

laurabaines committed:
WIP: Added a 'how to use' section

laurabaines committed:
WIP@ Changed to intended use

laurabaines committed:
WIP: Added usage and best practice intro

laurabaines committed:
WIP: Updated to include comments from BG

laurabaines committed:
WIP: Big update

Added headings, checked for consistency of 'media' and 'photo' / 'video'

laurabaines committed:
WIP: Added section on social media management (may need more work)

laurabaines committed:
WIP: Added non-compliance section

laurabaines committed:
WIP: Update to Non-Compliance

laurabaines committed:
Update README.md

laurabaines committed:
WIP: Rationale

Taking photographs and videos

laurabaines committed:
WIP: Most of rationale

laurabaines committed:
WIP: Updated the rationale

laurabaines committed:
WIP: Updated the rationale

Markdown is not displaying correctly on some of the indented rationales. At present, I cannot work out why as the formatting appears the same on every single one.

laurabaines committed:
WIP: Moved the markdown as it wasn't working indented

laurabaines committed:
WIP: Social Media Update 1 - Final

Final Amendments before first public release

laurabaines committed:
WIP: Amended layout errors

laurabaines committed:
WIP: Renamed #2

laurabaines committed:
WIP: Added a How to Contribute section

laurabaines committed:
WIP: Amended errors
